math/src/main/java/com/powsybl/math/matrix/SparseMatrix.java

```diff
@@ -385,6 +385,15 @@ protected int getEstimatedNonZeroValueCount() {
         return values.size();
     }
 
+    private native SparseMatrix transpose(int m, int n, int[] ap, int[] ai, double[] ax);
+
+    @Override
+    public SparseMatrix transpose() {
+        SparseMatrix transposed = transpose(rowCount, columnCount, columnStart, rowIndices.getData(), values.getData());
+        transposed.setRgrowthThreshold(rgrowthThreshold);
```

Comment:
Just noticed that we're setting the same r growth threshold to all matrix, which questions me about it being a parameter of SparseMatrix.

Reply:
We just consider that a derivated matrix (transposed, multiplied) has by default the same parameter value than the initial one which seems to be fine to me. The rgrowthThreshold can be defined in the SparseMatrixFactory, so all matrices created from the same factory have by default the same value.